app/src/forms/form/exportService.js

```diff
@@ -22,6 +22,16 @@ class SubmissionData extends Model {
           query.where('createdAt', '<=', maxDate);
         }
       },
+      filterDeleted(query, showDeleted) {
+        if(showDeleted == 'false') {
+          query.where('deleted', false);
+        }
+      },
+      filterDrafts(query, showDrafts) {
+        if(showDrafts == 'false') {
+          query.where('draft', false);
+        }
+      },
```

Comment:
This can be simplified to be more concise and leverage truthiness checks:
```js
filterDeleted(query, showDeleted) {
  if(!showDeleted) query.where('deleted', false);
},
filterDrafts(query, showDrafts) {
  if(!showDrafts) query.where('draft', false);
},
```
If we explicitly check for "false" instead of falsiness, we may miss a few edge cases such as when they're undefined (as they're not explicitly "false").

Reply:
We do this in a couple spots I think
```
      filterDraft(query, value) {
        if (value !== undefined) {
          query.where('draft', value);
        }
      },
      filterDeleted(query, value) {
        if (value !== undefined) {
          query.where('deleted', value);
        }
      },
 ```

But whatever works and is concise, we should look later at making these standard.

Also this model should go in the model area with the other ones (not sure why it was originally here) but that can go when we refactor all those to their own files.